client/Dockerfile-5.12

```diff
@@ -150,3 +150,5 @@ RUN if [ "$BUILD_QT" != "1" ] ; then echo Download precompiled Qt. && \
       tar -xvf ${QT_TARBALL} && \
       rm ${QT_TARBALL} \
     ; fi
+
```

Comment:
I guess you didn't want to keep that line and the next? (they appear alone in the jq commit)

Reply:
Intended, wanted to have a clear separator for whatever will come after the Qt-related stuff ;)

Reply:
Ah OK, fair enough.